### PR TITLE
Update CONTRIBUTING.md to allow posting proposals before help wanted label

### PR DESCRIPTION
**Pull request by iwiznia**
### Details
See https://expensify.slack.com/archives/C01GTK53T8Q/p1681404821427459

No screenshots, no checklist and no tests, since this is just an md update

### CODE REVIEW

**Review by allroundexperts — DISMISSED** [review]
Looks awesome!

File: contributingGuides/CONTRIBUTING.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Additionally if you want to discuss an idea with the open source community witho
 3. If you cannot reproduce the problem, pause on this step and add a comment to the issue explaining where you are stuck or that you don't think the issue can be reproduced.
 
 #### Propose a solution for the job
-4. Do not propose solutions to jobs without the `Help Wanted` label applied. Any proposals submitted when that label is not present will not be reviewed.
+4. If you propose solutions to jobs without the Help Wanted label applied, know that the proposals will not be reviewed till the label is added and that we might never add the label, thus never hire a contributor
```
Comment (puneetlath) [inline contributingGuides/CONTRIBUTING.md:114]: How about: 
>You can propose solutions on any issue at at any time, but if you propose solutions to jobs before the `Help Wanted` label applied, you do so at your own risk. Proposals will not be reviewed until the label is added and there is always a chance that we might not add the label or hire an external contributor for the job.

Comment (iwiznia) [reply contributingGuides/CONTRIBUTING.md:114]: Sounds good, next time please use the suggest feature, so I can click to merge it 😄

**Comment by iwiznia** [comment]
Updated

File: contributingGuides/CONTRIBUTING.md (hunk 1 above)
Comment (puneetlath) [reply contributingGuides/CONTRIBUTING.md]: Yep yep. My b!

File: contributingGuides/CONTRIBUTING.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Additionally if you want to discuss an idea with the open source community witho
 3. If you cannot reproduce the problem, pause on this step and add a comment to the issue explaining where you are stuck or that you don't think the issue can be reproduced.
 
 #### Propose a solution for the job
-4. Do not propose solutions to jobs without the `Help Wanted` label applied. Any proposals submitted when that label is not present will not be reviewed.
+4. You can propose solutions on any issue at at any time, but if you propose solutions to jobs before the `Help Wanted` is label applied, you do so at your own risk. Proposals will not be reviewed until the label is added and there is always a chance that we might not add the label or hire an external contributor for the job.
```
Comment (puneetlath) [inline contributingGuides/CONTRIBUTING.md:114]: Sorry, typo!

```suggestion
4. You can propose solutions on any issue at at any time, but if you propose solutions to jobs before the `Help Wanted` label is applied, you do so at your own risk. Proposals will not be reviewed until the label is added and there is always a chance that we might not add the label or hire an external contributor for the job.
```

**Review by puneetlath — CHANGES_REQUESTED** [review]

**Comment by iwiznia** [comment]
Updated agai

**Review by puneetlath — APPROVED** [review]

**Pull request merged**